services/plugin/src/Plugin.tsx

```diff
@@ -179,15 +179,9 @@ export const Plugin = ({
                     console.error(err)
                 })
         }
-    }, [
-        memoizedPropsToPass,
-        inErrorState,
-        // The following should be pretty stable:
-        alertsAdd,
-        clientWidth,
-        height,
-        width,
-    ])
+        // Skip `width` and `height` props to avoid updates when changing size:
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [memoizedPropsToPass, inErrorState, alertsAdd, clientWidth])
```

Comment:
Won't this cause issues if width/height is provided and later removed (or vice versa)? The callbacks won't be sent to the child, even though it should switch to child-driven sizing.

Reply:
Yes, it will -- it's not great, but I felt okay doing this because this is how it was before, too

That's also probably an unlikely case; someone would be best off using a `className` or a `container` for changeable styles like that

Reply:
The boolean works 👌 added that!